src/hooks/useReportUppyInstance.js

```diff
@@ -16,7 +16,7 @@ export default function useReportUppyInstance(reportType) {
   fileRef.current = files;
 
   useEffect(() => {
-    const uppyInstance = Uppy({
+    const uppyInstance = new Uppy({
```

Comment:
Uppy instances also appear in ProfileUploader.jsx ln.20 , UploadManager.jsx ln 32, SettingsFileUpload.jsx ln. 39,

Reply:
Thanks!